wandb/sdk/artifacts/artifact.py

```diff
@@ -1138,13 +1154,22 @@ def add_file(
                 to the basename of the file.
             is_tmp: If true, then the file is renamed deterministically to avoid
                 collisions.
+            skip_cache: If set to `True`, W&B will not copy/move files to the cache while uploading
```

Comment:
```suggestion
            skip_cache: If set to `True`, W&B will not copy files to the cache after uploading.
```
We only ever write to the cache after uploading; this setting shouldn't affect staging files prior to that.